packages/graph/hash_graph/tests/integration/postgres/mod.rs

```diff
@@ -414,11 +392,11 @@ impl DatabaseApi<'_> {
         let roots = subgraph
             .roots
             .into_iter()
-            .filter_map(|edition_id| subgraph.vertices.remove(&edition_id))
-            .map(|vertex| {
-                let Vertex::KnowledgeGraph(vertex) = vertex else { unreachable!() };
-                let KnowledgeGraphVertex::Entity(persisted_entity) = *vertex;
-                persisted_entity
+            .filter_map(|edition_id| match edition_id {
+                GraphElementEditionId::Ontology(_) => None,
+                GraphElementEditionId::KnowledgeGraph(edition_id) => {
+                    subgraph.vertices.entities.remove(&edition_id)
```

Comment:
Just a question about the removal here. I assume it mutates the subgraph, is this intentional? I see we did it before too

Reply:
Just to highlight, this is in the integration tests, and the return type of this function is `Entity`, so the `Subgraph` is dropped

Reply:
Oh of course, my bad!